libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.spec.ts

```diff
@@ -393,7 +404,12 @@ describe('Toggle switch component', () => {
       expect(ngModel.touched).toEqual(false);
       expect(testComponent.modelValue).toEqual(true);
 
-      SkyAppTestUtility.fireDomEvent(buttonElement, 'blur');
+      if (buttonElement) {
+        SkyAppTestUtility.fireDomEvent(buttonElement, 'blur');
+      } else {
+        fail('Expected button element to exist');
```

Comment:
?

Reply:
These are all like this because `fireDomEvent` doesn't allow for `undefined` and because we decided to do `| null` and use `?` everywhere I had to be able to handle these cases somehow.

Reply:
I updated `fireDomEvent` to handle `null` or `undefined` and now have removed the if statement